bolt/lib/Core/DebugData.cpp

```diff
@@ -177,6 +177,13 @@ uint64_t DebugRangesSectionWriter::getSectionOffset() {
   return SectionOffset;
 }
 
+void DebugRangesSectionWriter::updateRangeBuffer(std::unique_ptr<DebugBufferVector> &CUBuffer) {
+  for(auto DebugInfo : *CUBuffer){
```

Comment:
Use Ranges Stream. See DebugRangeListsSectionWriter::finalizeSection() for example.